rando/src/main.rs

```diff
@@ -23,16 +23,49 @@ struct Opt {
     seed: Option<String>,
 }
 
-fn shuffle_area(
+/*
+   Returns a list of chests that are allowed to be randomized (so no crypt keys, crystal balls or medallions are returned here)
+   All these chests are piled on one big heap, randomized and passed into write_new_chests_for_area.
+*/
```

Comment:
I'm pretty sure it is idomatic in Rust to prefer line comments of block comments.  I've been sticking to that elsewhere in the project.